Added fix for missing CSS import that was breaking diff views

sgnn7 opened the pull request:
Diffs didn't have the proper CCS loaded so diffs were getting mangled on
opening and this change just adds that in.

[endlessm/eos-shell#6280]

sgnn7 committed:
Added fix for missing CSS import that was breaking diff views

Diffs didn't have the proper CCS loaded so diffs were getting mangled on
opening and this change just adds that in.

dbnicholson commented:
Thanks for finding that. The sourcediff is also used on the package view screen via src/webui/app/views/package/rdiff.rhtml. See for example https://obs-master.endlessm-sf.com/package/rdiff?linkrev=base&package=gstreamer1.0&project=eos%3Amaster%3Acore&rev=19. What would be the right way to get the stylesheet used there, too?

sgnn7 committed:
More fixes for broken CSS in diffs

This now fixes pacakgediff to use the proper CSS class and make it not
look ugly.
